src/components/Trilemma/index.tsx

```diff
@@ -33,49 +24,44 @@ const Trilemma = () => {
   }
 
   return (
-    <Flex
-      alignItems="center"
-      flexDirection={{ base: "column", lg: "row" }}
-      justifyContent="space-between"
-      columnGap="12"
-      rowGap="8"
-    >
-      <Stack
-        spacing="8"
-        mt={{ base: "16", md: "20" }}
-        mb={{ lg: "20" }}
-        ms={{ md: "12" }}
-        me={{ md: "12", lg: 0 }}
-        flex={{
-          lg: "0 1 500px",
-        }}
-      >
-        <Heading fontSize="2rem">
+    <Flex className="flex-col items-center justify-between space-y-8 lg:flex-row lg:space-x-12 lg:space-y-0">
+      <Stack className="mt-16 space-y-8 md:mx-12 md:mt-20 lg:mb-20 lg:mr-0 lg:flex-[0_1_500px]">
+        <h2 className="text-4xl font-bold">
           {t("page-roadmap-vision-trilemma-h2")}
-        </Heading>
-        <Flex flexDir="column" gap="6">
-          <Text>{t("page-roadmap-vision-trilemma-p")}</Text>
-          <Text>{t("page-roadmap-vision-trilemma-p-1")}</Text>
-          <Text>{t("page-roadmap-vision-trilemma-p-2")}</Text>
-          <Text fontWeight={600} hideFrom="lg">
+        </h2>
+        <VStack className="space-y-6">
+          <p>{t("page-roadmap-vision-trilemma-p")}</p>
+          <p>{t("page-roadmap-vision-trilemma-p-1")}</p>
+          <p>{t("page-roadmap-vision-trilemma-p-2")}</p>
+          <p className="font-semibold lg:hidden">
             {t("page-roadmap-vision-trilemma-modal-tip")}:
-          </Text>
-        </Flex>
+          </p>
+        </VStack>
         <Card {...cardDetail} minH="300px" hideBelow="lg" mt="6" />
       </Stack>
-      <Hide above="lg">
-        <Drawer
-          isOpen={mobileModalOpen}
-          onClose={handleModalClose}
-          placement="bottom"
-        >
-          <DrawerOverlay background="rgba(0,0,0,0.3)" />
-          <DrawerContent borderTopRadius="2xl" background="background.base">
-            <Card {...cardDetail} background="none" border="none" my="8" />
-            <DrawerCloseButton top="6" insetInlineEnd="6" />
-          </DrawerContent>
-        </Drawer>
-      </Hide>
+      <Sheet open={mobileModalOpen} onOpenChange={handleModalClose}>
```

Comment:
We need to add a condition to only display/use the Sheet in mobile devices. Currently I can see it on desktop <https://deploy-preview-14102--ethereumorg.netlify.app/en/roadmap/vision/>

![image](https://github.com/user-attachments/assets/968e13e1-4af3-4da6-bda5-49b44a76ba41)